frontend/src/lib/lemon-ui/LemonSelect/LemonSelect.tsx

```diff
@@ -143,6 +144,7 @@ export function LemonSelect<T extends string | number | boolean | null>({
                 sideIcon={isClearButtonShown ? <div /> : undefined}
                 type="secondary"
                 status="stealth"
+                aria-disabled={isItemsEmpty}
```

Comment:
`LemonButton` does not have a `aria-disabled` prop, because it handles this HTML attribute internally based on the `disabledReason` prop